### PR TITLE
Added Standard Titan icon to the compiled exe

### PR DESCRIPTION
**Pull request by rishav394**
## Description

Added an Icon so that the compiled `Titan.exe` will have the standard titan logo from /Titan/Resources/Logo.ico

## Type

```
Feature
```

## Checklist

*Check the checkbox(es) that apply.*

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) guide.
- [x] `./build.sh` is passing with every build step and every test locally.
- [x] `./build.ps1` is passing with every build step and every test locally.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I tested it on **Linux** and on **Windows** and it's working as intended.
- [x] I tested it on **Windows** and it's working as intended.

## Discussion

Needs to be tested for exe's icon on `linux`.
`./build.sh` needs to be tested too.

## Reviewers

@Marc3842h, ...

### CODE REVIEW

File: tools/NetHook2/README.md [hunk 1]
```diff
@@ -1,27 +0,0 @@
-# NetHook 2
```
Comment (Marc3842h) [inline tools/NetHook2/README.md]: The build script removed the contents in the `tools` directory. Please revert this change.

File: TitanTest/TitanTest.csproj [hunk 2]
```diff
@@ -119,6 +119,9 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```
Comment (Marc3842h) [inline TitanTest/TitanTest.csproj]: There is no reason to add a service.

File: Titan/Titan.csproj [hunk 3]
```diff
@@ -36,6 +36,9 @@
   <PropertyGroup Condition="'$(OS)' == 'Unix'">
     <DefineConstants>__UNIX__</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup>
+    <ApplicationIcon>Logo.ico</ApplicationIcon>
```
Comment (Marc3842h) [inline Titan/Titan.csproj]: I'd love to have it reference the `Logo.ico` in the `Resources` directory instead of having two files in different places that are the same.

File: Titan/Titan.csproj [hunk 4]
```diff
@@ -225,6 +228,7 @@
   </ItemGroup>
   <ItemGroup>
     <Content Include="accounts.json" />
+    <Content Include="Logo.ico" />
```
Comment (Marc3842h) [inline Titan/Titan.csproj]: Same as above.

**Comment by raspbianlike** [comment]
Apparently this will only affect Windows users, Linux users will still have to specify their own launcher with the icon.
